README.md

```diff
@@ -106,17 +106,24 @@ You can use this simple function to add and remove the animations:
 
 ```javascript
 function animateCSS(element, animationName, callback) {
-    const node = document.querySelector(element)
-    node.classList.add('animated', animationName)
+    const node = document.querySelector(element);
```

Comment:
Do we really need all those `;`? It's 2019 and it's pretty safe to omit those.

Reply:
Agree